tests/modules/asset_groups/resources/test_create_asset_group.py

```diff
@@ -369,96 +427,6 @@ def test_create_bulk_asset_group(flask_app_client, researcher_1, test_root, db):
         tus_utils.cleanup_tus_dir(transaction_id)
 
 
-def test_create_asset_group_simulate_detection(
-    flask_app_client, researcher_1, contributor_1, internal_user, test_root, db
-):
-    # pylint: disable=invalid-name
-    import uuid
-    from tests.modules.asset_groups.resources.utils import TestCreationData
-
-    transaction_id, test_filename = tus_utils.prep_tus_dir(test_root)
-    asset_group_uuid = None
-    try:
-        data = TestCreationData(transaction_id)
-        data.add_filename(0, test_filename)
-        resp = asset_group_utils.create_asset_group(
-            flask_app_client, researcher_1, data.get()
-        )
-        asset_group_uuid = resp.json['guid']
-        assert 'assets' in resp.json
-        assets = resp.json['assets']
-        assert len(assets) == 1
-        asset_guid = assets[0]['guid']
-        assert 'asset_group_sightings' in resp.json
-        asset_group_sighting_uuid = resp.json['asset_group_sightings'][0]['guid']
-        job_id = str(uuid.uuid4())
-        path = f'sighting/{asset_group_sighting_uuid}/sage_detected/{job_id}'
-        data = {
-            'response': {
```

Comment:
@hwindsor this is the actual response from sage. Next time please don't change the test. Now the code is broken.

Reply:
The reason why this is different is because this is not an API call, this is a callback so the response isn't the same

Reply:
(I have no problems with test refactoring, it just can't change the test content)